src/contracts/Model/Suggestion/ContentSuggestion.php

```diff
@@ -0,0 +1,69 @@
+<?php
+
+/**
+ * @copyright Copyright (C) Ibexa AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+declare(strict_types=1);
+
+namespace Ibexa\Contracts\Search\Model\Suggestion;
+
+use Ibexa\Contracts\Core\Repository\Values\ContentType\ContentType;
+
+final class ContentSuggestion extends Suggestion
+{
+    private int $contentId;
```

Comment:
Use full API Content object. You already have it available in `SearchHitToContentSuggestionMapper`.